tests/test_permissions.py

```diff
@@ -23,127 +25,172 @@
         },
     }
 )
+PERMISSIONV2 = PermissionV2(
+    **{
+        "name": "test_permission",
+        "repo": {
+            "repositories": ["test_repository"],
+            "actions": {
+                "users": {"test_user": ["read", "annotate", "write", "delete",]},
+                "groups": {"developers": ["read", "annotate", "write", "delete",],},
+            },
+        },
+        "includePatterns": ["**"],
+        "excludePatterns": [],
+    }
+)

```

Comment:
Now that we know that the JSON changed, we need to update it in the tests too!
```suggestion
PERMISSIONV2 = PermissionV2(
    **{
        "name": "test_permission",
        "repo": {
            "repositories": ["test_repository"],
            "actions": {
                "users": {"test_user": ["read", "annotate", "write", "delete",]},
                "groups": {"developers": ["read", "annotate", "write", "delete",],},
            },
        },
        "include-patterns": ["**"],
        "exclude-patterns": [],
    }
)
```